proteus/Comm.py

```diff
@@ -2,10 +2,24 @@
 Module for controlling MPI
 """
 
+import ctypes
+import imp
+import os
 import sys
+
 import petsc4py
 from .Profiling import logEvent as log
 
+try:
```

Comment:
I just copied/pasted over from parun, I think the warning will still show up for any config files where the preload_libs environment variable is not specified.  But at least it's in a more reasonable place now.